### PR TITLE
Bug 1645657, added note about IBM Power issue

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1645657

@sferich888 PTAL

### CODE REVIEW

File: release_notes/ocp_3_10_release_notes.adoc [hunk 1]
```diff
@@ -2171,6 +2171,30 @@ link:https://access.redhat.com/errata/RHBA-2018:2869[RHBA-2018:2869] advisory.
 The container images included in the update are provided by the
 link:https://access.redhat.com/errata/RHBA-2018:2708[RHBA-2018:2708] advisory.
 
+
+[IMPORTANT]
+====
+There is a known issue in which some components of {product-title} can not
+function because images for IBM Power have not yet been built. These images
```
Comment (kalexand-rh) [inline release_notes/ocp_3_10_release_notes.adoc]: s/Power/POWER

**Comment by ahardin-rh** [comment]
@kalexand-rh  Thank you for your feedback!
@anpingli Can you please provide a QE review?

**Comment by anpingli** [comment]
LGTM

**Comment by ahardin-rh** [comment]
@anpingli Thank you!

**Pull request merged**